examples/dotnet/CMakeLists.txt.in

```diff
@@ -25,4 +25,5 @@ add_subdirectory(PdxAutoSerializer)
 add_subdirectory(PdxSerializableCs)
 add_subdirectory(PutGetRemove)
 add_subdirectory(RemoteQueryCs)
+add_subdirectory(TransactionCs)
```

Comment:
What does `Cs` mean?

Reply:
@pivotal-jbarrett , it's C#. Not sure how it started.

Reply:
Turns out, you can’t have two projects in the same VS solution whose names only differ by case.  We (I) adopted the convention so we could still build all examples in the same solution.

Reply:
The configure and install targets use paths based on the cmake variable ADD_EXAMPLE_NAME, so the names have to match for now.  We could extricate the two things from each other, but that's IMO work beyond the scope of this PR.